patchback/config.py

```diff
@@ -30,6 +31,13 @@ class PatchbackConfig:
     )
     """Prefix that the older/stable version branch has."""
 
+    failed_label_prefix: str | None = attr.ib(default=DEFAULT_FAILED_BACKPORT_LABEL_PREFIX)
```

Comment:
TODO: Add a check that `failed_label_prefix` isn't the same as `backport_label_prefix` to prevent infinite loops.